wrapper/fmriprep_docker.py

```diff
@@ -237,6 +237,7 @@ def __call__(self, parser, namespace, values, option_string=None):
 
     def _is_file(path, parser):
         """Ensure a given path exists and it is a file."""
+        path = os.path.abspath(path)
```

Comment:
I would suggest that we switch to `return os.path.abspath(path)`, so that the exception matches the user input.

Feel free to disregard and merge, if you disagree. This is a good fix.

Reply:
I'll merge it in as is, I feel the full path to the invalid license would be helpful when debugging